File: 1/solution.exs
File.stream!("input.txt")
|> Enum.map(&String.trim/1)
|> Enum.reduce(
  [0],
  fn
    "", sums ->
      [0 | sums]

    line, [sum | sums] ->
      {calories, _} = Integer.parse(line)
      [sum + calories | sums]
  end
)
|> Enum.sort(:desc)
  |> Enum.take(3)
  |> IO.inspect()
  |> Enum.sum()
  |> IO.inspect()

File: 1/solution.py
sums = [0]

with open("input.txt") as fp:
    for line in fp:
        match line.strip():
            case "":
                sums.append(0)
            case num:
                sums[-1] += int(num)

print(max(sums))
print(sum(sorted(sums)[-3:]))
